build.gradle

```diff
@@ -23,7 +23,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.amazonaws:aws-lambda-java-log4j2:1.4.0'
```

Comment:
Does aws-lambda-java-core itself have a dependency on log4j?

Reply:
It does not have any dependencies. 
https://github.com/aws/aws-lambda-java-libs/tree/master/aws-lambda-java-core

Here is the full dependency graph

```
./gradlew -q dependencies --configuration runtimeClasspath

------------------------------------------------------------
Root project - datadog-lambda-java
------------------------------------------------------------

runtimeClasspath - Runtime classpath of source set 'main'.
+--- com.amazonaws:aws-lambda-java-core:1.2.1
+--- com.amazonaws:aws-lambda-java-events:3.10.0
|    \--- joda-time:joda-time:2.6
+--- com.google.code.gson:gson:2.8.9
+--- org.apache.httpcomponents:httpclient:4.5.13
|    +--- org.apache.httpcomponents:httpcore:4.4.13
|    +--- commons-logging:commons-logging:1.2
|    \--- commons-codec:commons-codec:1.11
+--- com.squareup.okhttp3:okhttp:4.9.2
|    +--- com.squareup.okio:okio:2.8.0
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.0 -> 1.4.10
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.10
|    |    |    \--- org.jetbrains:annotations:13.0 -> 22.0.0
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.0 -> 1.4.10
|    \--- org.jetbrains.kotlin:kotlin-stdlib:1.4.10 (*)
+--- org.apache.commons:commons-lang3:3.12.0
+--- org.slf4j:slf4j-log4j12:1.7.32
|    +--- org.slf4j:slf4j-api:1.7.32
|    \--- log4j:log4j:1.2.17
+--- org.jetbrains:annotations:22.0.0
+--- io.opentracing:opentracing-api:0.33.0
+--- io.opentracing:opentracing-util:0.33.0
|    +--- io.opentracing:opentracing-api:0.33.0
|    \--- io.opentracing:opentracing-noop:0.33.0
|         \--- io.opentracing:opentracing-api:0.33.0
+--- com.datadoghq:dd-trace-api:0.90.0
\--- com.datadoghq:java-dogstatsd-client:3.0.0
     \--- com.github.jnr:jnr-unixsocket:0.36
          +--- com.github.jnr:jnr-ffi:2.1.16
          |    +--- com.github.jnr:jffi:1.2.23
          |    +--- org.ow2.asm:asm:7.1
          |    +--- org.ow2.asm:asm-commons:7.1
          |    |    +--- org.ow2.asm:asm:7.1
          |    |    +--- org.ow2.asm:asm-tree:7.1
          |    |    |    \--- org.ow2.asm:asm:7.1
          |    |    \--- org.ow2.asm:asm-analysis:7.1
          |    |         \--- org.ow2.asm:asm-tree:7.1 (*)
          |    +--- org.ow2.asm:asm-analysis:7.1 (*)
          |    +--- org.ow2.asm:asm-tree:7.1 (*)
          |    +--- org.ow2.asm:asm-util:7.1
          |    |    +--- org.ow2.asm:asm:7.1
          |    |    +--- org.ow2.asm:asm-tree:7.1 (*)
          |    |    \--- org.ow2.asm:asm-analysis:7.1 (*)
          |    +--- com.github.jnr:jnr-a64asm:1.0.0
          |    \--- com.github.jnr:jnr-x86asm:1.0.2
          +--- com.github.jnr:jnr-constants:0.9.17
          +--- com.github.jnr:jnr-enxio:0.30
          |    +--- com.github.jnr:jnr-constants:0.9.17
          |    \--- com.github.jnr:jnr-ffi:2.1.16 (*)
          \--- com.github.jnr:jnr-posix:3.0.61
               +--- com.github.jnr:jnr-ffi:2.1.16 (*)
               \--- com.github.jnr:jnr-constants:0.9.17

(*) - dependencies omitted (listed previously)

```